api/app/controllers/webhook.controller.js

```diff
@@ -0,0 +1,53 @@
+const db = require("../sequelize/models");
+const User = db.user;
+const Review = db.review;
+const Transcript = db.transcript;
+const Op = db.Sequelize.Op;
+
+
+exports.create = async (req, res) => {
+    // const action = req.headers["x-github-event"];
+    const pull_request = req.body;
+     const action = pull_request.action;
+
+
+    // Check if it's a pull request event
+const PR_EVENTS = {
```

Comment:
@Extheoisah resolved on error. This is a javascript file; "as const" won't work. A better approach is to declare each const separately maybe in another file, which I did already.

Reply:
Oh! that is correct. I have gotten so used to typescript, LOL.

Reply:
> Oh! that is correct. I have gotten so used to typescript, LOL.

You can ignore my suggestion then